R/pdf_document.R

```diff
@@ -201,7 +202,7 @@ pdf_document <- function(toc = FALSE,
 # Use filter to set pdf geometry defaults (while making sure we don't override
 # any geometry settings already specified by the user)
 pdf_pre_processor <- function(metadata, input_file, runtime, knit_meta, files_dir,
-                              output_dir) {
+                              output_dir, extra_dependencies) {
```

Comment:
`extra_dependencies` should not be an argument of the pre-processor; you can safely leave it out here and it the `extra_dependencies` argument of `pdf_document` can be passed to the inside of this processor